tools/src/bin/output-file.rs

```diff
@@ -324,7 +324,7 @@ fn test_interpolate_coverage() {
             vec![-1, 1, -1, 1, -1],
             vec![-1, 1, -2, 1, -1]
         ],
-        // don't interpolate if the whole file is uncoveredj
+        // don't interpolate if the whole file is uncovered
```

Comment:
?

Reply:
This and some of the others may have been a function of focus-follows-mouse when using a wired-mouse where the wire acts as a spring, allowing the focus to veer into an editor window accidentally.  I generally try to use wireless mouses these days for this reason! :)

Reply:
I once had a bunch of colleagues who used that. And there was a short period when I used an x-mouse thing on Windows for this. I've long ago abandoned that. I do have one coworker who uses it. I get to periodically poke fun at him.

But, I understand the allure.